packages/components/src/core/DropdownMenu/style.ts

```diff
@@ -115,132 +46,31 @@ export const StyledHeaderTitle = styled("div", {
 
     return `
       font-family: ${typography?.fontFamily};
-      color: black;
-      padding-top: ${spacings?.xxs}px;
-      padding-right: ${spacings?.s}px;
-      padding-left: ${spacings?.s}px;
-      ${search ? `padding-bottom: 0px;` : `padding-bottom: ${spacings?.xs}px;`}
+      padding-right: ${spacings?.m}px;
+      margin-bottom: ${search ? spacings?.s : spacings?.m}px;
     `;
   }}
 `;
 
-const StyledDropdownMenuAutocompleteWrapperColumnStyles = (
-  props: StyleProps
-) => {
-  const { search, title } = props;
-  const spacings = getSpaces(props);
-
-  return `
-    .SdsAutocompleteMultiColumn-column-root{
-      padding-right: ${
-        search || title ? `${spacings?.xxs}px` : `${spacings?.m}px`
-      };
-
-      &:not(:last-child) {
-        margin-right: ${
-          search || title ? `2px` : `${spacings?.m}px`
-        } !important;
-        padding-right: ${search || title ? `0` : `${spacings?.m}px`} !important;
-      }
-
-      .MuiAutocomplete-popper,
-      .MuiPopper-root {
-        .MuiAutocomplete-paper,
-        .MuiPaper-root {
-          .MuiAutocomplete-listbox {
-            padding: ${
-              title || search
-                ? `0 ${spacings?.m}px ${spacings?.s}px ${spacings?.s}px`
-                : `0`
-            } !important;
-          }
-
-          .MuiAutocomplete-groupLabel {
-            padding: ${
-              title || search ? `0` : `${spacings?.xxs}px ${spacings?.s}px`
-            } !important;
-          }
-
-          .MuiAutocomplete-groupUl {
-            margin: ${
-              title || search
-                ? `0 0 ${spacings?.m}px`
-                : `0 ${spacings?.s}px ${spacings?.m}px`
-            } !important;
-          }
-        }
-      }
+export const StyledDropdownMenuAutocompleteWrapper = styled("div")`
+  & .MuiAutocomplete-popper,
```

Comment:
should we use MUI classes object here too?

Reply:
Fixed :D